gridstatus/pjm.py

```diff
@@ -1133,17 +1133,41 @@ def get_interconnection_queue(self, verbose=False):
         return queue
 
     @support_date_range(frequency=None)
-    def get_solar_forecast(self, date, end=None, verbose=False):
+    def get_solar_forecast(
```

Comment:
We aren't consistent in this across the codebase, but there's a question here whether we should use one method with an extra parameter or separate `get_solar_forecast_hourly` and `get_solar_forecast_5_min` methods. I'm in favor of separate methods to maintain a consistent interface with our `get_` methods.

Reply:
i agree. i think one method per source endpoint is highly preferable for maintenance. 

it's nice to know exactly what endpoint/dataset is going to hit without having to understand parameters. 

I wish we never combined endpoints for things like the `get_lmp` method

Reply:
sounds good, updated.